.github/workflows/ci.yml

```diff
@@ -6,7 +6,12 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-20.04, macos-latest, windows-latest]
-        dotnet: ['3.1.200']
+        dotnet: ['3.1.200', '5.0.8']
+        include:
+          - dotnet: '3.1.200'
+            framework: netcoreapp3.0
+          - dotnet: '3.0.8'
+            framework: net5
```

Comment:
also, is this right? why 3.0.8 maps to .NET5?

Reply:
ah that's a typo indeed, it should be 5.0.8

Reply:
Thanks, its typo